src/main/docker/artemis/Dockerfile

```diff
@@ -0,0 +1,68 @@
+#TODO: cache beforehand of the gradle dependencies which don't change often ...
+
+#-----------------------------------------------------------------------------------------------------------------------
+# build stage
+#-----------------------------------------------------------------------------------------------------------------------
+FROM ghcr.io/ls1intum/docker-jdk-node-yarn as builder
```

Comment:
Is there an advantage over using this image instead of the regular `docker.io/library/eclipse-temurin:17`?

Gradle should pull and not use the system NodeJS automatically anyway (see `gradle.properties` for the versions, `settings.gradle` for the node-gradle plugin). Then on updates it only has to be changed in one place instead of multiple and no separate Docker image has to be maintained.

When/If a cache for `.gradle/` as noted in the ToDo above is implemented, then that should/can be cached as well.

Reply:
I choose this for now to have the same build environment like on bamboo. I haven't yet dag more into why we use that specific image.